lib/rules/no-curly-component-invocation.js

```diff
@@ -47,14 +49,47 @@ const BUILT_INS = new Set([
 
 const ALWAYS_CURLY = new Set(['yield']);
 
+function toAngleAttrValue(attr) {
+  const node = attr.value;
+  if (node.type === 'SubExpression') {
+    if (node.path.original === 'concat') {
+      // eslint-disable-next-line unicorn/prefer-spread
+      return b.concat(
+        node.params.map((p) => {
+          if (p.type === 'PathExpression') {
+            return b.mustache(p);
+          } else if (p.type === 'StringLiteral') {
+            return b.text(p.original);
+          } else if (p.type === 'SubExpression') {
+            return b.mustache(p.path, p.params, p.hash);
+          } else {
+            return p;
```

Comment:
if we hit this `else`, we should avoid fixing

Reply:
@rwjblue could you provide `breaking` example?
in my mind else handle cases you described above:

```
{{#foo-bar 123}}{{/foo-bar}}
{{#foo-bar true}}{{/foo-bar}}
{{#foo-bar false}}{{/foo-bar}}
{{#foo-bar undefined}}{{/foo-bar}}
{{#foo-bar null}}{{/foo-bar}}
```